CMakeLists.txt

```diff
@@ -258,6 +266,7 @@ AddTarget(TYPE shared NAME GuiUtils OUTPUT_PREFIX FH
 AddTarget(TYPE shared NAME GuiWidgets OUTPUT_PREFIX FH
     SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/src/Gui/Widgets
     EXPORT_INCLUDES
+    INCLUDES ${CMAKE_CURRENT_SOURCE_DIR}/src/Gui/Widgets
```

Comment:
No, that should not be that way, I'd better fix AddTarget

Reply:
I added a fix to the mernel, can you rebase to the master branch and remove extra INCLUDES here?

Reply:
Yes, you are right